conf/mesos-site.xml

```diff
@@ -99,12 +109,12 @@
 
   <property>
     <name>mesos.hdfs.journalnode.cpus</name>
-    <value>1</value>
+    <value>0.5</value>
```

Comment:
If we leave them in, I'd rather we comment them out. That way, the user can still see the description (and default value), and know exactly how to add their own custom value for it. But at the same time we save on xml parsing time. I remember one of the first speedups that MapR made was clearing out the site.xml files, which actually shaved seconds off starting the various Hadoop components.